internal/services/integrationtesting/cert_test.go

```diff
@@ -180,10 +180,10 @@ func TestCertRotation(t *testing.T) {
 	).Complete(ctx)
 	require.NoError(t, err)
 
-	wait := make(chan struct{}, 1)
+	wait := make(chan error, 1)
 	go func() {
-		require.NoError(t, srv.Run(ctx))
-		wait <- struct{}{}
```

Comment:
We can't make this assertion inside the goroutine, so we instead return the error from the channel and assert about it.